scapy/contrib/automotive/uds.py

```diff
@@ -1454,6 +1454,20 @@ class UDS_NR(Packet):
         0x36: 'exceedNumberOfAttempts',
         0x37: 'requiredTimeDelayNotExpired',
         0x3A: 'secureDataVerificationFailed',
+        0x50: 'Certificate verification failed - Invalid Time Period',
+        0x51: 'Certificate verification failed - Invalid Signature',
+        0x52: 'Certificate verification failed - Invalid Chain of Trust',
+        0x53: 'Certificate verification failed - Invalid Type',
+        0x54: 'Certificate verification failed - Invalid Format',
+        0x55: 'Certificate verification failed - Invalid Content',
+        0x56: 'Certificate verification failed - Invalid Scope',
+        0x57: 'Certificate verification failed – Invalid Certificate (revoked)',
+        0x58: 'Ownership verification failed',
+        0x59: 'Challenge calculation failed',
+        0x5A: 'Setting Access Rights failed',
+        0x5D: 'Session key creation/derivation failed',
```

Comment:
0x5B??

Reply:
Thanks for pointing out. 
There is an update now.